style.css

```diff
@@ -365,3 +384,193 @@ textarea {
   letter-spacing: 0.03em;
   margin-left: 111px;
 }
+
+.d-none-m {
+  display: none;
+}
+
+@media screen and (min-width: 992px) {
+  .d-none {
+    display: none;
+  }
+
+  .two {
+    display: none;
+  }
+
+  .d-block {
+    display: block;
+  }
+
+  .header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .logo {
+    margin-left: 154px;
+  }
+
+  .nav-list {
+    display: flex;
+    gap: 20px;
+    margin-right: 156px;
+    color: #fff;
+    list-style: none;
+  }
+
+  .container {
+    background-image: url('./photos/desktop/head.png');
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right bottom;
+    height: 870px;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    align-items: center;
+    padding-right: 55%;
+    gap: 10%;
+  }
+
+  .list {
+    display: flex;
+    flex-direction: column;
+    margin-right: 50px;
+    margin-bottom: 12px;
+  }
+
+  .temp {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 24px;
+  }
+
+  .temp img {
+    width: 684px;
+    height: 448px;
+  }
+
+  .data {
+    margin-bottom: 147px;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: auto auto auto;
+  }
+
+  .common-1 {
+    background-image: url('./photos/desktop/common-1.png');
+  }
+
+  .bg-1 {
+    background-image: url('./photos/desktop/bg-1.png');
+  }
+
+  .common-3 {
+    background-image: url('./photos/desktop/common-3.png');
+  }
+
+  .container-2 {
+    background-image: url('./photos/desktop/footer.png');
+    background-position: 100% 136px;
+    margin-left: 142px;
+    background-repeat: no-repeat;
+    background-size: 39%;
+  }
+
+  .about {
+    width: 566px;
+  }
+
+  .text-res {
+    width: 457px;
+    height: 148px;
+    line-height: 25px;
+    font-size: 17px;
+  }
+
+  .skills {
+    display: flex;
+    gap: 20%;
+    flex-direction: row;
+  }
+
+  .hr {
+    width: 84%;
+    background-color: #fff;
+    height: 0;
+    border: 1px solid #fff;
+    margin-bottom: 90px;
+  }
+
+  .form {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    flex-direction: row;
+    gap: 25px;
+  }
+
+  .prg {
+    display: flex;
+    justify-content: center;
+    width: 334px;
+    height: 256px;
+    margin-bottom: 82%;
+  }
+
+  .list-2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  hr {
+    margin: 23px 0 34px 4px;
+  }
+
+  .hr-1 {
+    position: absolute;
+    height: 1px;
+    padding-left: 0;
+    padding-right: 0;
+    top: 0;
+  }
+
+  .line-work {
+    margin-top: 20%;
+    width: 100%;
+  }
+
+  .cards2 {
+    text-align: center;
+    display: flex;
+    padding-left: 13%;
+  }
+
```

Comment:
## [OPTIONAL]
- [ ] Instead of using padding-left to % you can use `px` or `rem`, this will eliminate the unwanted space at the right on the work section. :point-down:
![image](https://user-images.githubusercontent.com/65068771/149638747-cb2b02e2-c65f-4a6b-80b7-9ee0ed4fba77.png)